src/components/CarouselGallery/CarouselGallery.module.scss

```diff
@@ -32,16 +32,18 @@
 		transform-style: preserve-3d; /* needed even when not rotating for Chrome */
 		transform-origin: 50% 50% $offset; /* needed even when not rotating for Safari */
 		&.rotating {
-			animation: $tt rotate-carousel infinite linear;
+			animation: rotate-carousel infinite linear;
+			animation-duration: $rotation-period;
+			@media (prefers-reduced-motion: reduce) {
+				animation-duration: 2 * $rotation-period;
+			}
```

Comment:
I was under the impression that for users who prefer reduced motion, we would just not rotate the carousel at all, but this is fine, especially since it's probably easier this way.

Reply:
I suppose it could go either way, not sure what would be best.